docs/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+Version 0.40.0:
+
+- Library: Added `Adaptor` module and `ToJava` class for working with Java collections (thanks Stephen!)
```

Comment:
Do we want to publish this, given they depend on an unstable feature?

Reply:
Rest OK and spell check?